commitlintplugins.test.ts

```diff
@@ -84,6 +84,46 @@ test('prefer-slash-over-backslash2', () => {
     let commitMsgWithSlash = "foo/bar: bla bla bla";
     let preferSlashOverBackslash2 = runCommitLintOnMsg(commitMsgWithSlash);
     expect(preferSlashOverBackslash2.status).toBe(0);
+
+});
+
+
+test('header-max-length-with-suggestions1', () => {
+    let commitMsgWithThatExceedsHeaderMaxLength =
+        "foo: this is only a title with a configuration in it that exceeds header max length";
+    let headerMaxLength1 = runCommitLintOnMsg(commitMsgWithThatExceedsHeaderMaxLength);
+    let expected_message = `"configuration" -> "config"`
+    // console.log('HERE ==========>' + replacementSuggestion1.stdout)
+    expect(headerMaxLength1.status).not.toBe(0);
+    expect((headerMaxLength1.stdout + '').includes(expected_message)).toEqual(true)
+});
+
+
+test('header-max-length-with-suggestions2', () => {
+    let commitMsgWithThatExceedsHeaderMaxLength =
+        "foo: this is only a title with a absolute in it that exceeds header max length";
+    let headerMaxLength2 = runCommitLintOnMsg(commitMsgWithThatExceedsHeaderMaxLength);
+    let expected_message = `"absolute" -> "abs"`
```

Comment:
@realmarv instead of testing just another random string, let's modify this test to check that strings with a space also work, e.g. "1 second" -> "1sec"

Reply:
Done